magenta/models/melody_rnn/melody_rnn_generate.py

```diff
@@ -223,9 +229,21 @@ def run_with_flags(generator):
   # files.
   date_and_time = time.strftime('%Y-%m-%d_%H%M%S')
   digits = len(str(FLAGS.num_outputs))
+
   for i in range(FLAGS.num_outputs):
     generated_sequence = generator.generate(input_sequence, generator_options)
 
+    if FLAGS.exclude_primer_midi:
+      # Remove any notes with start_time less than start_time of generated
+      # sequence. Otherwise, subtract length of removed input_sequence from
+      # start_time and end_time of remaining notes.
+      for note in generated_sequence.notes[:]:
+        if note.start_time < start_time:
+          generated_sequence.notes.pop(0)
```

Comment:
@cghawthorne since `note` is already accessible, how about using `remove`:

`generated_sequence.notes.remove(note)`

When using an index, it seems we'll need to enumerate the `generate_sequence.notes` slice, like: 

```
      for index, note in enumerate(generated_sequence.notes[:]):
        if note.start_time < start_time:
          del generated_sequence.notes[index]
```